src/Talk/Create/CreateTalkRequest.php

```diff
@@ -2,43 +2,37 @@
 
 declare(strict_types=1);
 
-namespace App\Dto;
+namespace App\Talk\Create;
 
-use App\Entity\Talk;
+use App\Entity\Speaker;
+use App\Talk\Talk;
 use Symfony\Component\Validator\Constraints as Assert;
 
-final class TalkRequest
+class CreateTalkRequest
 {
     /**
+     * @var string|null
      * @Assert\Length(max=255)
      * @Assert\NotBlank()
      */
     public $title;
 
     /**
+     * @var string|null
      * @Assert\NotBlank()
      */
     public $description;
 
+    /** @var Speaker|null */
     public $speaker;
 
-    public static function createFromEntity(Talk $talk): TalkRequest
+    public static function createFromEntity(Talk $talk): self
```

Comment:
Not sure where that stands yet, but we have to make sure and enforce the same policy over all of the project.
I'll check later on, but if the class name is used elsewhere, I'd rather have one different PR changing them all.

Consistency is key here.